mesos-rxjava-core/src/main/java/org/apache/mesos/rx/java/MesosSchedulerClient.java

```diff
@@ -71,7 +75,7 @@ public MesosSchedulerClient(
     }
 
     @NotNull
-    public Observable<Protos.Event> openEventStream(@NotNull final Call subscription) {
+    public Observable<In> openEventStream(@NotNull final Out subscription) {
```

Comment:
This line seems backward to me. I would imagine having the method parameter be the `In` and `Observable<Out>` as the return.